static/styles/dark_theme.css

```diff
@@ -309,9 +309,7 @@ body.dark-theme {
         color: hsl(236, 33%, 90%);
     }
 
-    .recent_topics_container .unread_count,
-    .topic-list-item .unread_count,
-    .expanded_private_message .unread_count {
+    .unread_count {
```

Comment:
I changed it to directly use `.unread_count`.

Reply:
Cool, thanks :))